packages/sync/src/links/QueueLink.ts

```diff
@@ -32,10 +41,8 @@ export default class QueueLink extends ApolloLink {
     }
 
     public request(operation: Operation, forward: NextLink ) {
-        if (this.isOpen) {
-            return forward(operation);
-        }
-        if (operation.getContext().skipQueue) {
+        // TODO split this conditional and add a handler to notify of online only cases
+        if (this.isOpen || hasDirectives(["onlineOnly"], operation.query)) {
```

Comment:
constant this directive (no need for config - we need constants file to have some central piece where we put stuff that is indirectly connected to server.